pt/core-libraries/caching.rst

```diff
@@ -1,17 +1,614 @@
 Caching
 #######
 
+.. php:namespace::  Cake\Cache
+
+.. php:class:: Cache
+
+O cache pode ser usado para acelerar a leitura de recursos caros ou lentos, 
+mantendo uma segunda cópia dos dados necessários em um sistema de armazenamento 
+mais rápido ou mais próximo. Por exemplo, você pode armazenar os resultados de 
+consultas caras ou acesso remoto ao serviço da Web que não muda com freqüência 
+em um cache. Uma vez no cache, é muito mais barato do que acessar o recurso remoto.
+
+O armazenamento em cache no CakePHP é facilitado pela classe ``Cache``. Esta 
+classe fornece uma interface estática e uma API uniforme para interagir com 
+várias implementações de armazenamento em cache. O CakePHP fornece vários 
+mecanismos de cache e fornece uma interface simples se você precisar criar 
+seu próprio back-end. Os mecanismos de armazenamento em cache integrados são:
+
+* ``File`` O cache de arquivo é um cache simples que usa arquivos locais. 
+  É o mecanismo de cache mais lento e não fornece tantos recursos para operações 
+  atômicas. No entanto, como o armazenamento em disco geralmente é bastante barato, 
+  o armazenamento de objetos grandes ou elementos que raramente são gravados 
+  funciona bem em arquivos.
+* ``Memcached`` Usa a extensão `Memcached <http://php.net/memcached>`_.
+* ``Redis`` Usa a extensão `phpredis <https://github.com/nicolasff/phpredis>`_. O 
+  Redis fornece um sistema de cache rápido e persistente semelhante ao Memcached, 
+  também fornece operações atômicas.
+* ``Apcu`` O cache do APCu usa a extensão PHP `APCu <http://php.net/apcu>`_. Essa 
+  extensão usa memória compartilhada no servidor da web para armazenar objetos. 
+  Isso o torna muito rápido e capaz de fornecer recursos atômicos de leitura/gravação.
+* ``Wincache`` O Wincache usa a extensão `Wincache <http://php.net/wincache>`_. 
+  O Wincache é semelhante ao APC em recursos e desempenho, mas otimizado para 
+  Windows e IIS.
+* ``Array`` Armazena todos os dados em uma matriz. Esse mecanismo não fornece 
+  armazenamento persistente e deve ser usado em conjuntos de testes de aplicativos.
+* ``Null`` O mecanismo nulo não armazena nada e falha em todas as operações de leitura. 
+
+Independentemente do CacheEngine que você escolher, seu aplicativo interage com
+:php:class:`Cake\\Cache\\Cache`.
+
 .. _cache-configuration:
 
-Configuring Cache Class
-=======================
+Configuring Cache Engines
```

Comment:
This is all english. We don't want to have english in translation pages because it is never updated.